litestar/datastructures/headers.py

```diff
@@ -276,7 +279,20 @@ def to_header(self, include_header_name: bool = False) -> str:
 
         return (f"{self.HEADER_NAME}: " if include_header_name else "") + self._get_header_value()
 
+    def dict(
+        self,
+        exclude_none: bool = False,
+        exclude_empty: bool = False,
+        by_alias: bool = False,
+        exclude: Optional[set] = None,
+        **kwargs: bool,
+    ) -> dict[str, Any]:
```

Comment:
```suggestion
    ) -> Dict[str, Any]:
```
You can only use `dict[str, Any]` on python 3.9+, unless you add `from __future__ import annotations` at the top of the module, in which case all annotations will be strings at runtime. However, using that directive breaks your `cast_to_type()` method on `CacheControlHeader`, as `get_args('Optional[int]') == ()` (notice the string annotation).